.github/buildomat/jobs/clippy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#:
+#: name = "clippy (helios)"
+#: variety = "basic"
+#: target = "helios-latest"
+#: rust_toolchain = "nightly-2022-09-27"
+#: output_rules = []
+#:
+
+# Run clippy on illumos (not just other systems) because a bunch of our code
+# (that we want to check) is conditionally-compiled on illumos only.
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+# Put tools on our PATH to satisfy install_builder_prerequisites.
+export PATH="$PATH:$PWD/out/cockroachdb/bin:$PWD/out/clickhouse"
+banner prerequisites
+ptime -m bash ./tools/install_builder_prerequisites.sh -y
+
+banner clippy
+# See the corresponding GitHub Actions job for more about these arguments.
+ptime -m cargo +'nightly-2022-09-27' clippy --all-targets -- --deny warnings --allow clippy::style
```

Comment:
why does this one need explicit nightly version and the other doesn't?

Reply:
I don't think there's a good reason.  I went for consistency with the toolchain used elsewhere (and actually didn't notice we _weren't_ specifying a toolchain on the other one).